pypsa/variables.csv

```diff
@@ -27,3 +27,4 @@ StorageUnit,state_of_charge,False,False,False
 StorageUnit,p_nom,False,True,False
 StorageUnit,n_mod,False,False,True
 StorageUnit,spill,False,False,True
+StorageUnit,spill_cost,False,False,False
```

Comment:
```suggestion
StorageUnit,spill,False,False,True
```
To keep the same structure, the `variables.csv` should rather be extended by a column `spill_cost` which is only True for StorageUnit spill.